backend_py/src/lights/rpi_pwm_hardware.py

```diff
@@ -0,0 +1,60 @@
+from rpi_hardware_pwm import HardwarePWM, HardwarePWMException
+from .pwm_controller import PWMController
+from typing import Dict
+import logging
+
+class RPiHardwarePWMController(PWMController):
+    NAME = 'Hardware PWM Controller'
+
+    def __init__(self, chip=0, pins=None) -> None:
+        super().__init__()
+
+        self.pwm_supported = True
+
+        if pins is None:
+            pins = {
+                18: 0,
+                19: 1
+            }
+        self.PWM_PINS = pins
+
+        self.pwm_objects: Dict[int, HardwarePWM] = {}
+
+        for pin in self.PWM_PINS.keys():
+            try:
+                self.pwm_objects[pin] = HardwarePWM(pwm_channel=self.PWM_PINS[pin], hz=7812.5, chip=chip)
```

Comment:
`7812.5` is currently a Magic Number. Extract it to a constant and describe meaning in the variable name.

Reply:
`TESTED_FREQ` or smth